docs/source/guides/testing-react-apollo.md

```diff
@@ -0,0 +1,340 @@
+---
+title: Testing React Apollo
```

Comment:
We don't currently use the words "React Apollo" as a brand name in that order. Maybe "testing react with apollo" or "testing your frontend"?